### PR TITLE
#162163242 Show article read time

### PR DESCRIPTION
**Pull request by Nta1e**
#### What does this PR do?
Displays the time it takes to read a specific article.

#### Description of tasks to be completed?
Show article read time both on viewing all articles and viewing a specific article

#### How can this be manually tested?
- Navigate to the homepage where all articles are returned to see read time on every article
- View a specific article to view read time of that specific article
#### What are the relevant pivotal tracker stories?
(#162163242)[https://www.pivotaltracker.com/story/show/162163242]
screenshot 2019-01-30 at 19 01 16
screenshot 2019-01-30 at 19 00 31

### CODE REVIEW

**Review by bibangamba — APPROVED** [review]

**Review by KicaRonaldOkello — APPROVED** [review]

**Comment by Kasulejoseph** [comment]
This is good...

**Review by Kasulejoseph — APPROVED** [review]

**Review by kafuuma — APPROVED** [review]
This is good @Nta1e 
well done

**Review by JaymesKat — CHANGES_REQUESTED** [review]
exclude the `.idea/workspace.xml` from source control.

Otherwise looks good

File: src/components/articles/articleHeader/ArticleHeader.scss [hunk 1]
```diff
@@ -6,10 +6,13 @@
...
 }
-
 .btn.btn-sm {
   padding: 0.5rem 1.6rem;
   font-size: 7px;
   border-radius: 20px;
   size: 45px;
-}
\ No newline at end of file
+}
```
[the remark this answers is not in the document]
Comment (JaymesKat) [reply src/components/articles/articleHeader/ArticleHeader.scss]: @Nta1e Fix this

**Comment by JaymesKat** [comment]
@Nta1e Remove the few hound-ci errors in the .scss files

File: src/components/articles/articleHeader/ArticleHeader.scss [hunk 2]
```diff
@@ -6,10 +6,13 @@
   padding: 0.5rem 1.6rem;
   font-size: 0.54rem;
 }
-
 .btn.btn-sm {
   padding: 0.5rem 1.6rem;
   font-size: 7px;
   border-radius: 20px;
   size: 45px;
+}
```
[the remark this answers is not in the document]
Comment (JaymesKat) [reply src/components/articles/articleHeader/ArticleHeader.scss:14]: Fix this @Nta1e

File: src/components/articles/articleHeader/ArticleHeader.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (JaymesKat) [reply src/components/articles/articleHeader/ArticleHeader.scss:14]: Fix this too @Nta1e

**Review by JaymesKat — CHANGES_REQUESTED** [review]

**Review by JaymesKat — APPROVED** [review]
LGTM

**Pull request merged**